resources/migrations/030_add_field_visibility_type.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: 30
+      author: agilliland
+      changes:
+        - addColumn:
+            tableName: metabase_field
+            columns:
+              - column:
+                  name: visibility_type
+                  type: varchar(32)
+                  constraints:
+                    nullable: true
+                    deferrable: false
+                    initiallyDeferred: false
+        - addNotNullConstraint:
+            columnDataType: varchar(32)
+            columnName: visibility_type
+            defaultNullValue: unset
```

Comment:
Because I want the column to be `NOT NULL` because it's required.  This allows us to add the column and make it not null and provides a clear way for us to know the data is in transition, which is leveraged by the data migration.